src/list-data/request-actions.ts

```diff
@@ -1,63 +1,78 @@
-import { RequestCode } from '@/enums'
+import { NrRequestActionCodes } from '@/enums'
 import { RequestActionsI } from '@/interfaces/models'
 
+/** All the request actions - some are headers and others are selectable items. */
 export const RequestActions: RequestActionsI[] = [
   {
-    text: 'Register or Incorporate a',
+    isHeader: true,
+    group: 0,
+    text: 'For businesses that do not exist yet:'
+  },
+  {
+    group: 0,
+    text: 'Start a new BC-based business',
+    subtext: 'Incorporation or Registration',
     shortDesc: 'New Request',
-    value: RequestCode.NEW,
-    blurbs: `Create a new business in British Columbia or register a business you formed in another province or
-              territory, country or federal jurisdiction so that you may also conduct business here in BC.`
+    value: NrRequestActionCodes.NEW_BUSINESS
```

Comment:
The new structure is actually fairly similar to the old one -- it provides a Request Action Code that is used in the rest of the flows (eg, NEW, MVE, CHG, AML, CNV).